input/ig.xml

```diff
@@ -609,6 +609,14 @@
 			<description value="Deangelo Howe"/>
 			<exampleCanonical value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-patient"/>
 		</resource>
+		<resource>
```

Comment:
remove the term 'deidentified' - this is a use case not tackled, and you cannot declare no identifier and name to be 'deidentified'. Do not use a Patient resource as an example for suppressed data in AU Core until meaningful use case and design decisions have been made.